tests/test_filter_fields.py

```diff
@@ -22,15 +22,16 @@
 from mongo_connector.doc_managers.doc_manager_simulator import DocManager
 from mongo_connector.locking_dict import LockingDict
 from mongo_connector.oplog_manager import OplogThread
-from mongo_connector.test_utils import ReplicaSet, assert_soon, close_client
+from mongo_connector.test_utils import ReplicaSetSingle, assert_soon, \
+    close_client
```

Comment:
You can use `()` to do imports that span multiple lines, which pep8 recommends over the backslash.